### PR TITLE
Use default nixpkgs from iohk-nix

### PR DESCRIPTION
**Pull request by jbgi**
since it is now also using 20.09 channel.

### CODE REVIEW

File: nix/sources.json [hunk 1]
```diff
@@ -41,22 +41,10 @@
         "homepage": null,
         "owner": "input-output-hk",
         "repo": "iohk-nix",
-        "rev": "83109208047b07d5f0c103c87e6685c61c36a9f6",
-        "sha256": "1wfflfhd3jc8j49kgbvw0absvibkwm77wrjjc9ls8xkrcnprn4l7",
+        "rev": "72262b212e0cd8efe119466f0cb123f9d3760d35",
```
Comment (rvl) [inline nix/sources.json:44]: Would you be able to use latest master, which contains a commitIdFromGitRepo improvement?

**Review by rvl — APPROVED** [review]
Looks good. I also tried this with cardano-wallet and it works fine.

File: nix/sources.json (hunk 1 above)
Comment (jbgi) [reply nix/sources.json:44]: done.

**Review by dcoutts — APPROVED** [review]

**Comment by dcoutts** [comment]
bors merge

**Pull request merged**